cclib/parser/orcaparser.py

```diff
@@ -959,6 +959,27 @@ def energy_intensity(line):
             self.etoscs = numpy.array(etoscs)
             self.transprop[name] = (self.etenergies, self.etoscs)
 
+        if line.strip() == "CD SPECTRUM":
+            # -------------------------------------------------------------------
+            #                              CD SPECTRUM
+            # -------------------------------------------------------------------
+            # State   Energy Wavelength       R         MX        MY        MZ   
+            #         (cm-1)   (nm)       (1e40*cgs)   (au)      (au)      (au)  
+            # -------------------------------------------------------------------
+            #    1   43167.6    231.7      0.00000   0.00000  -0.00000   0.00000
+            #
+            # Presumably the transition energies have already been parsed.
```

Comment:
Should there be a warning if the energies haven't already been parsed? Or perhaps be more aggressive with an assert?

Reply:
Did you want to do this Eric?

Reply:
It's here: https://github.com/cclib/cclib/pull/684/files#diff-8d648f11426b9efdbf41dbfe1b5135b9R987